package.json

```diff
@@ -3,7 +3,10 @@
   "version": "3.0.0",
```

Comment:
@lukastaegert based on the conversation, this will be a major bump, correct? Feel free to make that change to `4.0.0`.

Reply:
Done